lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -866,14 +866,17 @@ def test_static_tab_list_get_invalid_course(self):
         self.assertEqual(response.status_code, 404)
 
     def test_static_tab_detail_get(self):
-        test_uri = self.base_courses_uri + '/' + self.test_course_id + '/static_tabs/syllabus'
+        # get static tab by tab name
+        test_uri = self.base_courses_uri + '/' + self.test_course_id + '/static_tabs/Static Tab'
```

Comment:
I think you should create new static tab with + sign in its name to test urlencode/decode is working as expected.